content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,627 @@
+---
+title: "Cold Start"
+description: "Describes the *cold start* process"
+weight: 2
+tags:
+- cold start
+- aws-vault
+- geodesic
+---
+
+Here we describe the `cold start` process, when we start with just one master AWS account and provision infrastructure for different environments.
+
+# Prerequisites
+
+* Choose an AWS region in which to provision all the resources - we use `us-west-2` for our reference architectures
+
+* Select the parent DNS domain name for your infrastructure - in these examples we use `cloudposse.co`
+
+* Choose the namespace for resource naming. We recommend using your company name or abbreviation as the namespace (e.g. `cloudposse`, `cp`, `cpco`) - we use `cpco` in all the examples
+
+* We assume you already have an AWS account. If not create an account at `https://aws.amazon.com`. This account will be the `root`
+
+* Login to the `root` account with the root credentials and do the following:
+  * Create new IAM group `admin`
+  * Assign `AdministratorAccess` policy to the group
+  * Create an IAM user with the name `admin`
+  * Add the user to the group
+  * Enable MFA for the user (we recommend using Google Authenticator as Virtual MFA device)
+  * Generate `Access Key ID` and `Secret Access Key` for the user (you'll need them in the next steps)
+
+* Install and setup [aws-vault](https://github.com/99designs/aws-vault) to store IAM credentials in your operating system's secure keystore and then generate temporary credentials from those to expose to your shell and applications
+
+You can install manually from [aws-vault](https://github.com/99designs/aws-vault/releases). 
+
+On MacOS, you may use `homebrew cask`:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Install aws-vault with brew" %}}
+```shell
+brew cask install aws-vault
+```
+{{% /dialog %}}
+
+Then setup your secret credentials in `aws-vault` in the `cpco` profile (input the IAM `Access Key ID` and `Secret Access Key` when prompted)
+
+{{% dialog type="code-block" icon="fa fa-code" title="Setup aws-vault" %}}
+```shell
+export AWS_VAULT_BACKEND=file
+aws-vault add cpco
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+Replace the profile name `cpco` with your own (for consistency, we recommend using the same name as the namespace in the Terraform modules).
+
+You should set `AWS_VAULT_BACKEND=file` in your shell `rc` config (e.g. `~/.bashrc`) so it persists.
+{{% /dialog %}}
+
+* Keep in mind that AWS has limits on the number of accounts in an organization (see [Organization reference limits](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_reference_limits.html)).
```

Comment:
This bullet point seems misplaced. Did you mean to include it in the dialog?

Reply:
it's one of the `Prerequisites` items.
moved it a few items up.